### ISSUE TITLE
2022 - remove Request-Resource from prefetch in all request tests

### ISSUE THREAD

**Issue by bryantaustin13** [cqframework/opioid-cds-r4#259]
The prefetch items in pagecontent/request example requests have this:
"item1": {  "response": {    "status": "200 OK"  },  "resource": {

It needs to be removed from all prefetches, along with the trailing “}”.
It is no longer needed and is incorrect.

Was OPD-98

**Comment by brynrhodes** [comment]
Blocked on Tooling PR: https://github.com/cqframework/cqf-tooling/pull/409

**Comment by c-schuler** [comment]
Need to check tooling for duplication bug

**Comment by alackerbauer** [comment]
fixed here: https://github.com/cqframework/cqf-tooling/pull/439, tested with opioid files by calling JAVA -jar tooling-cli-2.5.0-SNAPSHOT.jar -RollTestsDataDates -v=r4 -ip="path/to/opioid/file/to/test"with a jar files packaged from the change in #439. no nesting is happening, the dates are rolled correctly, only the order of the JSON prefetch items change since JsonObject is unordered. See example diff here: https://www.diffchecker.com/xaJD9u2U/

**Comment by alackerbauer** [comment]
a bug in the tooling when rolling the dates caused the entire prefetch-resource to be duplicated. The pullrequest #308 had the dates rolled before committing, so it containes invalide JSON files with duplicated nested resources.

I suggest deleting #308 and using #317 instead. I compared which files changed in #308 and #317 and the only difference is, that in #317 the following files are not changed:
* input/pagecontent/requests/HelloWorld/HelloWorld.json --> empty prefetch element was removed entirely in #308.
* input/pagecontent/requests/OpioidCDSREC08OrderSign/noPrefetch.json --> empty prefetch element was removed entirely in #308.
* files in input/examples --> there has never been any prefetch element, only formatting has changed.
